app/serializers/dependents_serializer.rb

```diff
@@ -1,6 +1,74 @@
 # frozen_string_literal: true
 
+START_EVENTS = %w[EMC SCHATTB].freeze
+LATER_START_EVENTS = %w[SCHATTB].freeze
+END_EVENTS = %w[T18 SCHATTT].freeze
+FUTURE_EVENTS = (LATER_START_EVENTS + END_EVENTS).freeze
+
+module DependentsHelper
+  def max_time(a, b)
+    a[:award_effective_date] <=> b[:award_effective_date]
```

Comment:
This will return -1, 0, or 1. Is that appropriate for where `max_time` is used in this file?

Reply:
I'm not sure? It seems to do what I need when tested

Reply:
This was just pulled out into a method because it was the same comparison used in multiple lines